### ISSUE TITLE
[Bug]: 

### ISSUE THREAD

**Issue by MuteTiefling** [FTBTeam/FTB-Mods-Issues#1240]
### Mod

FTB Quests

### Mod version

ftb-quests-neoforge-2100.1.0

### Forge / Fabric version

NeoForge - 21.0.20

### What issue are you having?

Tooltips show behind quests

![image](https://github.com/FTBTeam/FTB-Mods-Issues/assets/9543430/b6746e37-d454-4812-9aa6-420bffff2d07)

### Steps to reproduce

1. Make a quest
2. Add a reward or task
3. Mouse over the reward or task, the tooltip will appear behind the quest

**Comment by desht** [comment]
Yeah, there are some z-level issues in 1.21... vanilla's changed how things work a bit. Thought I'd accounted for everything, but here's another case :laughing:
